packages/node-core/src/db/db.module.ts

```diff
@@ -23,6 +32,42 @@ const DEFAULT_DB_OPTION: DbOption = {
   username: process.env.DB_USER ?? 'postgres',
   password: process.env.DB_PASS ?? 'postgres',
   database: process.env.DB_DATABASE ?? 'postgres',
+  ssl: process.env.DB_SSL === 'true' ?? false,
+};
+
+/** Adding the ssl option to the sequelize option. If the ssl is enabled ,
+USER should provide the path as environment variable
+- DB_CA_CERT: path to the CA file (Required)
+- DB_KEY_CERT: path to the KEY file (Optional)
+- DB_CLIENT_CERT: path to the CERT file (Optional)
+@return the option with ssl option
+@exception throw error if the ssl is enabled but there is not correct file path
+*/
+const dbOptionWithSSLOptions = (option: DbOption): DbOption => {
+  if (option.ssl) {
+    const ca_content = readFileSync(process.env.DB_CA_CERT).toString();
```

Comment:
Can you please add some error handling for this.

Reply:
> 

Hi @stwiname ,
It is great suggestion, I will work on the modification as per advice and together with the error handling

Reply:
introduced error handling by logging error, but the file reading content is common function for both  node-core and query package, so introduced the new function in common package and leveraged that for both node-core and query.